src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rate = 40;
+  let total = 0;
```

Comment:
Be more specific with variable names
```suggestion
  const dailyRate = 40;
  let totalPrice = 0;
```